docker/kopia-build/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM golang:1.17-alpine3.14 AS builder
```

Comment:
Looks like the new image is available now - [golang:1.19.1-alpine3.16](https://hub.docker.com/layers/library/golang/1.19.1-alpine3.16/images/sha256-8074e76a21f9e52ceec00387bdf357cc1caac37f1e31f1e312dc103e9f5e45d3?context=explore)